RescheduleAppointments/force-app/main/default/lwc/aaRebookingCalendar/aaRebookingCalendar.html

```diff
@@ -0,0 +1,252 @@
+<template>
```

Comment:
What is the "aa" in aaRebookingCalendar?

Reply:
It's for appointment assistant where the code is taken from. Originally it was meant to consume the actual component but since those are not yet developed on core I guess I should rename the component for the samples project.